x-pack/plugin/security/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecuritySmokeIT.java

```diff
@@ -55,11 +56,19 @@ public void testRemoteAccessPortFunctions() throws Exception {
         } else {
             // Check that we can search the fulfilling cluster from the querying cluster
             Request searchRequest = new Request("GET", "/my_remote_cluster:test_idx/_search");
-            Response response = client().performRequest(searchRequest);
-            assertOK(response);
-            ObjectPath responseObj = ObjectPath.createFromResponse(response);
-            int totalHits = responseObj.evaluate("hits.total.value");
-            assertThat(totalHits, equalTo(1));
+
+            // Since authentication is not wired up on the FC side, getting a 401 response is sufficient
+            // for testing connection (port, SSL) is working
+            final ResponseException e = expectThrows(ResponseException.class, () -> client().performRequest(searchRequest));
+            assertThat(e.getResponse().getStatusLine().getStatusCode(), equalTo(401));
+            assertThat(e.getMessage(), containsString("missing authentication credentials"));
+
+            // TODO: uncomment once authentication is wired up on the FC side
+            // Response response = client().performRequest(searchRequest);
+            // assertOK(response);
+            // ObjectPath responseObj = ObjectPath.createFromResponse(response);
+            // int totalHits = responseObj.evaluate("hits.total.value");
+            // assertThat(totalHits, equalTo(1));
```

Comment:
This bit depends on the FC side authentication wiring (#93412)
@n1v0lg Depending on which PR goes in first, one of us will need to amend it. 🙏